experiments.json

```diff
@@ -238,5 +238,20 @@
       "threshold": 0.2,
       "description": "Edit Top Sites is enabled"
     }
+  },
+  "pocketStories": {
+    "name": "Trending stories provided by Pocket",
+    "active": true,
+    "description": "Display recommendations of trending stories by Pocket",
+    "control": {
+      "value": true,
+      "description": "Pocket trending stories active"
+    },
+    "variant": {
+      "id": "exp-017-pocketstories",
+      "value": true,
+      "threshold": 0,
```

Comment:
I'd personally be ok with turning on for 20%, but @csadilek was thinking maybe we should wait until they get the sign-off from pocket folks on monday